modules/network-observability-enriched-flows-kafka.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: Procedure
+[id="network-observability-enriched-flows-kafka_{context}"]
+= Export enriched network flow data
+
+You can send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elastic, or Fluentd,
+
+.Prerequisites
+* Installed Kafka
+* The `FlowCollector` `spec.agent` is set to `Kafka`.
+
+.Procedure
+
+. Edit the `FlowCollector` to configure `spec.exporters` as follows:
++
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+exporters:
+    - type: KAFKA
+      kafka:
+        address: "kafka-cluster-kafka-bootstrap.netobserv"
+        topic: netobserv-flows-export   <1>
```

Comment:
I think there are extra spaces here to remove (yaml is picky about that). The dash from `- type` should be at the same column than the `n` from `name: cluster` above